cli/tsc/99_main_compiler.js

```diff
@@ -385,7 +385,7 @@ delete Object.prototype.__proto__;
   // paths must be either relative or absolute. Since
   // analysis in Rust operates on fully resolved URLs,
   // it makes sense to use the same scheme here.
-  const ASSETS = "asset:///";
+  const ASSETS_URL_PREFIX = "asset:///";
```

Comment:
We were supposed to change it to `deno:///assets/`, didn't we?

Reply:
Not yet and I’m not sure yet. Need to investigate more